core/src/main/java/kafka/server/share/SharePartition.java

```diff
@@ -1599,15 +1655,17 @@ && checkForStartOffsetWithinBatch(inFlightBatch.firstOffset(), inFlightBatch.las
                 }
             }
 
-            if (!stateBatches.isEmpty() && !isWriteShareGroupStateSuccessful(stateBatches)) {
-
-                // Even if write share group state RPC call fails, we will still go ahead with the state transition.
-                log.error("Failed to write the share group state on acquisition lock timeout for share partition: {}-{} memberId: {}. " +
-                                "Proceeding with state transition.", groupId, topicIdPartition, memberId);
+            if (!stateBatches.isEmpty()) {
```

Comment:
While you are at this file, there is an existing typo `acquistionLockTimerTask`.

Reply:
Corrected, thanks.